nncf/torch/pruning/export_helpers.py

```diff
@@ -467,9 +492,26 @@ def input_prune(cls, model: NNCFNetwork, node: NNCFNode, graph: NNCFGraph):
                              ' {}.'.format(node.data['key'], old_num_clannels, new_num_channels))
 
 
+@PT_PRUNING_OPERATOR_METATYPES.register('linear')
```

Comment:
I think you should implement similar logic for `def input_prune(...)` and `def output_prune(...)` to what that conv operation has.

Tests for Linear layer when it is in SE block and in the end of the model are required.

Reply:
Do we planning to make some tests for  `ModelPruner`? Now we have no any tests for it

Reply:
@AlexKoff88, @vshampor I suggest to remove ModelPruner and all `input_prune` and `output_prune` methods due to they are redundant now

Reply:
@daniil-lyakhov Agreed, but you should do it in a separate PR

Reply:
Please watch #945